Monika After Story/game/definitions.rpy

```diff
@@ -5153,17 +5158,23 @@ init 2 python:
         if not isinstance(timekeeper, datetime.datetime):
             timekeeper = datetime.datetime.combine(timekeeper, datetime.time())
 
+        #If we converted from datetime.date to datetime.datetime, this is already inexact
+        #so we only check this if we'd originally passed in a timekeeper with a datetime.datetime
+        elif inexact:
+            timekeeper = timekeeper.replace(hour=0, minute=0, second=0, microsecond=0)
```

Comment:
if a `date` is passed in instead of a `datetime`, we already set the time portion to 0, so we could just pass in the date portion of a datetime rather than add an extra parameter. 

I don't see the usefullness of the inexact parameter. If we want inexact, pass in dates. If we want precise, pass in datetimes.